gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -135,8 +135,10 @@ typedef struct _GstTensorFilterProperties
   int latency; /**< The average latency over the recent 10 inferences in microseconds */
   int throughput; /**< The average throughput in the number of outputs per second */
   int invoke_dynamic; /**< True for supporting invoke with flexible output. */
-
   uint32_t suspend; /**< Timeout (ms) for suspend. (Unload the framework) */
+
+  void (*invoke_async_callback) (void* async_handle, GstTensorMemory *output); /**< Callback function to handle multiple async outputs */
```

Comment:
Please consider using typedef for the invoke_async_callback function pointer to improve code readability and maintainability. This would simplify its declaration within GstTensorFilterProperties and nnstreamer_filter_enable_invoke_async().

Reply:
Okay, I will use typedef, thank you.